packages/web/src/components/search/DataSearch.js

```diff
@@ -1059,9 +1059,8 @@ class DataSearch extends Component {
 		const {
 			theme, themePreset, size, recentSearchesIcon, popularSearchesIcon,
 		} = this.props;
-		const hasSuggestions = currentValue
-			? suggestionsList.length || this.topSuggestions.length
-			: this.defaultSuggestions.length;
+		const hasSuggestions
```

Comment:
@mohdashraf010897 Instead of changing the condition here, we should update the getter method for `defaultSuggestions`. The current change would create the side effects because `suggestionList` and `topSuggestions` can have stale values.